src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2092,11 +2094,19 @@ public void AarContentExtraction ([Values (false, true)] bool useAapt2)
 				Assert.IsTrue (builder.Build (proj), "Build should have succeeded");
 				Assert.IsTrue (builder.Output.IsTargetSkipped ("_ResolveLibraryProjectImports"),
 					"_ResolveLibraryProjectImports should not have run.");
+
+				var doc = XDocument.Load (cache);
+				var expectedCount = doc.Elements ("Paths").Elements ("ResolvedResourceDirectories").Count ();
+
 				aar.Timestamp = DateTimeOffset.UtcNow.Add (TimeSpan.FromMinutes (2));
 				Assert.IsTrue (builder.Build (proj), "Build should have succeeded");
 				Assert.IsFalse (builder.Output.IsTargetSkipped ("_ResolveLibraryProjectImports"),
 					"_ResolveLibraryProjectImports should have run.");
 
+				var doc = XDocument.Load (cache);
```

Comment:
I think this a compiler error:

```suggestion
				doc = XDocument.Load (cache);
```